runtime/runtime-domain-service/java/build.gradle

```diff
@@ -15,37 +15,21 @@
  *
  */
 
-plugins {
-    id 'base'
-    id 'jacoco-report-aggregation'
-}
-
-repositories {
-    mavenCentral()
-}
-
-
-reporting {
-    reports {
-        testCodeCoverageReport(JacocoCoverageReport) {
-            testType = TestSuiteType.UNIT_TEST
-        }
-    }
-}
-
-tasks.named('check') {
-    dependsOn tasks.named('testCodeCoverageReport', JacocoReport)
-}
+apply from: "$projectDir/../../../common-gradle-scripts/java.gradle"
 
 tasks.register('java_build') {
     group 'Java build'
-    println "build java"
+    doLast {
+        println "build java"
+    }
 }
 
 tasks.named('build').configure {
     dependsOn java_build
 }
 
-subprojects {
-    apply from: "$projectDir/../../common-gradle-scripts/java.gradle"
+dependencies {
+    implementation 'org.wso2.apk:org.wso2.apk.apimgt.impl:0.1.0-SNAPSHOT'
```

Comment:
Shall we add those also to the versions toml?

Reply:
This will fix once admin-ds changes added.